### ISSUE TITLE
 Aria-selected and aria-checked are not output correctly for trees

### ISSUE THREAD

**Issue by kolaps33** [FreedomScientific/standards-support#432]
### Summary

Aria-selected state is not narrated to the user.

**For more and more detailed info please see comment from JAWS below 
https://github.com/FreedomScientific/VFO-standards-support/issues/432#issuecomment-686385093**

Repro steps:
1. Navigate to the URL:
 https://fluentsite.z22.web.core.windows.net/maximize/tree-multiselect-example-shorthand/false
2. Navigate inside the tree and observe the narration of item which can be selected
Expected: JAWS narrates information about aria-selected state 
Actual: JAWS doesn't narrate anything about "selected" or "not selected" state
![tree-item-aria-selected](https://user-images.githubusercontent.com/22620150/91851240-21b85c00-ec5f-11ea-9f23-00ce2e6fe301.PNG)

#### JAWS version and build number

2020.20006.12 ILM
 
#### Operating System and version
Win 10 Enterprise
10.0.19041 Build 19041

#### Browser and version:
Chrome
Version 85.0.4183.83 (Official Build) (64-bit)

#### Comment
Based on the aria aria-selected should be supported or treeitem and option role based on documentation:
https://www.w3.org/TR/wai-aria-1.1/#aria-selected

**Comment by JAWS-test** [comment]
The reason is in my opinion, that normally the focused entry is identical to the selected entry, so an additional output of "selected" would be redundant. This is not the case when using aria-multiselectable=true, so this ARIA attribute will output the status of aria-selected correctly.

**Comment by kolaps33** [comment]
Hi @JAWS-test 
I un`

**Comment by JAWS-test** [comment]
I checked aria-multiselectable on your listbox example, not on the tree example. Your new tree example has no aria-multiselectable in the source code. The status of the checkboxes within the treeitems should be transmitted with aria-checked: https://www.w3.org/TR/wai-aria-1.2/#aria-checked

**Comment by kolaps33** [comment]
Hi @JAWS-test  :)
About the tree example, we took usage of "aria-selected" from the site:
https://www.w3.org/TR/wai-aria-practices-1.1/#tree_roles_states_props

> if the tree supports multiple selection:
> All selected nodes have aria-selected set to true.
> All nodes that are selectable but not selected have aria-selected set to false.
> If the tree contains nodes that are not selectable, those nodes do not have the aria-selected state.

**Comment by JAWS-test** [comment]
of course, JAWS should also support aria-selected in trees if multiple selection is possible. This was not the case in your example. If aria-selected in trees with multiple selection is not supported, this is a JAWS bug. In your example, it is still better to use aria-checked instead of aria-selected because only aria-checked allows the value mixed, as your example requires

**Comment by JAWS-test** [comment]
Please also note that the ARIA specification regarding attributes and trees is not final and is currently under discussion: https://github.com/w3c/aria/issues/1311

**Comment by JAWS-test** [comment]
I have checked the example https://www.w3.org/TR/wai-aria-practices-1.1/examples/treeview/treeview-1/treeview-1a.html with JAWS 2020 and current version of Chrome:
* aria-selected=true/false is not output in the tree, even if the tree has aria-multiselectable=true. This is a JAWS error
* aria-checked=true/false is correctly output in the tree
* aria-checked=mixed is output as "not activated partially activated", which is a JAWS error.

Note: In the example the attributes aria-selected, aria-checked and aria-multiselectable are missing. I have added them manually

**Comment by kolaps33** [comment]
@JAWS-test thank you very much for clarification and pointing us to the discussion. 

I renamed the bug as you suggest. 
As well I removed listbox mention from description, avoid confusion.

**Comment by elbazitay** [comment]
Hi, there's any estimation when this issue will be closed?

**Comment by kolaps33** [comment]
validated with example:
https://www.w3.org/WAI/ARIA/apg/patterns/treeview/examples/treeview-1b/

with JAWS version 2023.2303.144 and I hear JAWS started narration of aria-selected state:

![image](https://github.com/FreedomScientific/standards-support/assets/22620150/bc0ce0a1-b0e9-4214-b96f-d65fd6201674)
